addons/payment_stripe/controllers/main.py

```diff
@@ -19,9 +18,45 @@
 
 class StripeController(http.Controller):
     _checkout_return_url = '/payment/stripe/checkout_return'
+    _onboarding_return_url = '/payment/stripe/onboarding/return/'
+    _onboarding_refresh_url = '/payment/stripe/onboarding/refresh/'
     _validation_return_url = '/payment/stripe/validation_return'
     WEBHOOK_AGE_TOLERANCE = 10*60  # seconds
 
+    @http.route(
+        _onboarding_return_url + '<int:acquirer_id>/<int:menu_id>', type='http', auth='public'
+    )
+    def stripe_onboarding_return(self, acquirer_id, menu_id):
```

Comment:
Following a comment above, you should have changed these params to be GET params and thus kwargs with `None` as the default value. I saw that you choose `payment.payment_acquirer_menu` to be the fallback menu in `payment_acquirer.py`. You can now move that fallback assignation in this method since it needs a fallback again.

Reply:
correct :+1: